packages/experimental/AppearanceAdditions/ios/FRNAppearanceAdditions.m

```diff
@@ -103,11 +97,31 @@ + (BOOL)requiresMainQueueSetup {
     return @[ @"appearanceChanged" ];
 }
 
+- (dispatch_queue_t)methodQueue
```

Comment:
can we rename this to be more obvious like ui thread or main thread?

Reply:
Do you mean the name "methodQueue"? This is an overridden method from RCTBridgeModule so don't think I can change the method name

Reply:
I think this has to stay as is, yeah.